src/config.ts

```diff
@@ -18,7 +20,13 @@ const config: ApplicationConfiguration = {
         username: process.env.FES_DB_USER as string,
         password: process.env.FES_DB_PASSWORD as string,
         connectionString: process.env.FES_DB_CONNECTIONSTRING as string
-    }
+    },
+    session: {
+        cookieName: process.env.COOKIE_NAME as string,
```

Comment:
Have you added these to chs-configs?

Reply:
They already exist in chs-configs harmon1/global_env